### PR TITLE
OBSDOCS-352: document-new-silences-panel-in-monitoring-dev-console

### PR DESCRIPTION
**Pull request by bburt-rh**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OBSDOCS-352

Link to docs preview: https://63944--docspreview.netlify.app/openshift-enterprise/latest/monitoring/managing-alerts#managing-silences_managing-alerts

QE review: @juzhao 
- [ ] QE has approved this change.

Additional information: This PR updates the Managing silences section of the monitoring docs with content for the new Observe > Silences tab in the Developer perspective of the OCP web console.

This PR also edits existing content to make it accurate with the current UI and to bring the wording into alignment with the IBM Style guidance for documenting web UI elements.

Note: While doing work for this issue, I discovered that the modular docs guidelines are not being followed by the existing procedure module files in the "Managing silences" section of the monitoring docs: All of the proc modules here technically contain multiple procedures. Because fixing these issues is out of scope and would be extensive and intrusive for this PR, @bergerhoffer and I agree that these changes can be fixed in a separate PR. I've written up https://issues.redhat.com/browse/OBSDOCS-493 to fix the mod docs issues in the near future.

### CODE REVIEW

**Comment by kyoto** [comment]
Thanks @bburt-rh. Looks great!

LGTM

File: modules/monitoring-editing-silences.adoc [hunk 1]
```diff
@@ -6,30 +6,27 @@
...
 To edit a silence in the *Administrator* perspective:
 
-. Navigate to the *Observe* -> *Alerting* -> *Silences* page.
+. Go to *Observe* -> *Alerting* -> *Silences*.
 
-. For the silence you want to modify, select the {kebab} in the last column and choose *Edit silence*.
+. For the silence you want to modify, click {kebab} and select *Edit silence*.
 +
-Alternatively, you can select *Actions* -> *Edit Silence* in the *Silence Details* page for a silence.
+Alternatively, you can click *Actions* and select *Edit Silence* on the *Silence Details* page for a silence.
```
Comment (juzhao) [inline modules/monitoring-editing-silences.adoc:19]:
```suggestion
Alternatively, you can click *Actions* and select *Edit silence* on the *Silence details* page for a silence.
```
see picture: Screenshot from 2023-09-01 08-36-38

File: modules/monitoring-editing-silences.adoc [hunk 2]
```diff
@@ -6,30 +6,27 @@
...
 
-. In the *Edit Silence* page, enter your changes and select *Silence*. This will expire the existing silence and create one with the chosen configuration.
+. On the *Edit Silence* page, make changes and click *Silence*. Doing so expires the existing silence and creates one with the updated configuration.
```
Comment (juzhao) [inline modules/monitoring-editing-silences.adoc:21]:
```suggestion
. On the *Edit silence* page, make changes and click *Silence*. Doing so expires the existing silence and creates one with the updated configuration.
```
Screenshot from 2023-09-01 08-40-20

File: modules/monitoring-editing-silences.adoc [hunk 3]
```diff
@@ -6,30 +6,27 @@
...
+. Go to *Observe* -> *<project_name>* -> *Silences*.
 
-. Expand the details for an alert by selecting *>* to the left of the alert name. Select the name of the alert in the expanded view to open the *Alert Details* page for the alert.
-
-. Select the name of a silence in the *Silenced By* section in that page to navigate to the *Silence Details* page for the silence.
-
-. Select the name of a silence to navigate to its *Silence Details* page.
+. For the silence you want to modify, click {kebab} and select *Edit silence*.
++
+Alternatively, you can click *Actions* and select *Edit Silence* on the *Silence Details* page for a silence.
```
Comment (juzhao) [inline modules/monitoring-editing-silences.adoc:29]:
```suggestion
Alternatively, you can click *Actions* and select *Edit silence* on the *Silence details* page for a silence.
```

File: modules/monitoring-editing-silences.adoc [hunk 4]
```diff
@@ -6,30 +6,27 @@
...
 
-. Select *Actions* -> *Edit Silence* in the *Silence Details* page for a silence.
+. On the *Edit Silence* page, make changes and click *Silence*. Doing so expires the existing silence and creates one with the updated configuration.
```
Comment (juzhao) [inline modules/monitoring-editing-silences.adoc:31]:
```suggestion
. On the *Edit silence* page, make changes and click *Silence*. Doing so expires the existing silence and creates one with the updated configuration.
```

File: modules/monitoring-expiring-silences.adoc [hunk 5]
```diff
@@ -16,22 +16,22 @@ Expired silences older than 120 hours are garbage collected.
...
-. Navigate to the *Observe* -> *Alerting* -> *Silences* page.
+. Go to *Observe* -> *Alerting* -> *Silences*.
 
-. For the silence you want to modify, select the {kebab} in the last column and choose *Expire silence*.
+. For the silence or silences you want to expire, select the checkbox in the corresponding row.
+
+. Click *Expire silence*.
 +
-Alternatively, you can select *Actions* -> *Expire Silence* in the *Silence Details* page for a silence.
+Alternatively, to expire a single silence you can click *Actions* and select *Expire Silence* on the *Silence Details* page for a silence.
```
Comment (juzhao) [inline modules/monitoring-expiring-silences.adoc:27]:
```suggestion
Alternatively, to expire a single silence you can click *Actions* and select *Expire silence* on the *Silence details* page for a silence.
```
see the above attached picture

File: modules/monitoring-expiring-silences.adoc [hunk 6]
```diff
@@ -16,22 +16,22 @@ Expired silences older than 120 hours are garbage collected.
...
 
-. Select the name of a silence in the *Silenced By* section in that page to navigate to the *Silence Details* page for the silence.
+. For the silence or silences you want to expire, select the checkbox in the corresponding row.
 
-. Select the name of a silence to navigate to its *Silence Details* page.
-
-. Select *Actions* -> *Expire Silence* in the *Silence Details* page for a silence.
+. Click *Expire silence*.
++
+Alternatively, to expire a single silence you can click *Actions* and select *Expire Silence* on the *Silence Details* page for a silence.
```
Comment (juzhao) [inline modules/monitoring-expiring-silences.adoc:37]:
```suggestion
Alternatively, to expire a single silence you can click *Actions* and select *Expire silence* on the *Silence details* page for a silence.
```

File: modules/monitoring-expiring-silences.adoc [hunk 7]
```diff
@@ -16,22 +16,22 @@ Expired silences older than 120 hours are garbage collected.
...
-To expire a silence in the *Administrator* perspective:
+To expire a silence or silences in the *Administrator* perspective:
 
-. Navigate to the *Observe* -> *Alerting* -> *Silences* page.
+. Go to *Observe* -> *Alerting* -> *Silences*.
 
-. For the silence you want to modify, select the {kebab} in the last column and choose *Expire silence*.
+. For the silence or silences you want to expire, select the checkbox in the corresponding row.
+
+. Click *Expire silence*.
```
Comment (juzhao) [inline modules/monitoring-expiring-silences.adoc:25]: see picture below, if you select 1 silence, the button is "Expire 1 silence", 
Screenshot from 2023-09-01 08-51-22
select 2 silences, the button is "Expire 2 silences", and so on
Screenshot from 2023-09-01 08-53-24

File: modules/monitoring-expiring-silences.adoc [hunk 8]
```diff
@@ -16,22 +16,22 @@ Expired silences older than 120 hours are garbage collected.
...
-. Expand the details for an alert by selecting *>* to the left of the alert name. Select the name of the alert in the expanded view to open the *Alert Details* page for the alert.
+. Go to *Observe* -> *<project_name>* -> *Silences*.
 
-. Select the name of a silence in the *Silenced By* section in that page to navigate to the *Silence Details* page for the silence.
+. For the silence or silences you want to expire, select the checkbox in the corresponding row.
 
-. Select the name of a silence to navigate to its *Silence Details* page.
-
-. Select *Actions* -> *Expire Silence* in the *Silence Details* page for a silence.
+. Click *Expire silence*.
```
Comment (juzhao) [inline modules/monitoring-expiring-silences.adoc:35]: same as https://github.com/openshift/openshift-docs/pull/63944/files#r1312423029

File: modules/monitoring-silencing-alerts.adoc [hunk 9]
```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
 
 * In the *Administrator* perspective:
 
-. Navigate to the *Observe* -> *Alerting* -> *Alerts* page of the {product-title} web console.
+. Go to *Observe* -> *Alerting* -> *Alerts* in the {product-title} web console.
 
-. For the alert that you want to silence, select the {kebab} in the right-hand column and select *Silence Alert*. The *Silence Alert* form will appear with a pre-populated specification for the chosen alert.
+. For the alert that you want to silence, click {kebab} and select *Silence Alert* to open the *Silence Alert* page with a default configuration for the chosen alert.
```
Comment (juzhao) [inline modules/monitoring-silencing-alerts.adoc:32]:
```suggestion
. For the alert that you want to silence, click {kebab} and select *Silence alert* to open the *Silence alert* page with a default configuration for the chosen alert.
```

File: modules/monitoring-silencing-alerts.adoc [hunk 10]
```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
...
-
-. To create the silence, select *Silence*.
+. To save the silence, click *Silence*.
 
 * In the *Developer* perspective:
 
-. Navigate to the *Observe* -> *<project_name>* -> *Alerts* page in the {product-title} web console.
+. Go to *Observe* -> *<project_name>* -> *Alerts* in the {product-title} web console.
+
+. If necessary, expand the details for the alert by selecting *>* next to the alert name.
```
Comment (juzhao) [inline modules/monitoring-silencing-alerts.adoc:47]: as I checked on developer console, the details are expanded, not collapsed. keep this description is also fine

File: modules/monitoring-silencing-alerts.adoc [hunk 11]
```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
...
+
+. Click the alert message in the expanded view to open the *Alert Details* page for the alert.
```
Comment (juzhao) [inline modules/monitoring-silencing-alerts.adoc:49]:
```suggestion
. Click the alert message in the expanded view to open the *Alert details* page for the alert.
```

File: modules/monitoring-silencing-alerts.adoc [hunk 12]
```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
...
+
+. Click *Silence Alert* to open the *Silence Alert* page with a default configuration for the alert.
```
Comment (juzhao) [inline modules/monitoring-silencing-alerts.adoc:51]:
```suggestion
. Click *Silence alert* to open the *Silence alert* page with a default configuration for the alert.
```

File: modules/monitoring-silencing-alerts.adoc [hunk 13]
```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
...
+. To save the silence, click *Silence*.
 
-. Expand the details for an alert by selecting *>* to the left of the alert name. Select the name of the alert in the expanded view to open the *Alert Details* page for the alert.
+To silence a set of alerts by creating a silence configuration in the *Administrator* perspective:
 
-. Select *Silence Alert*. The *Silence Alert* form will appear with a prepopulated specification for the chosen alert.
+. Go to *Observe* -> *Alerting* -> *Silences* in the {product-title} web console.
 
-. Optional: Modify the silence.
+. Click *Create Silence*.
```
Comment (juzhao) [inline modules/monitoring-silencing-alerts.adoc:66]:
```suggestion
. Click *Create silence*.
```

File: modules/monitoring-silencing-alerts.adoc [hunk 14]
```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
...
 
-. You must add a comment before creating the silence.
+. On the *Create Silence* page, set the schedule, duration, and label details for an alert.
```
Comment (juzhao) [inline modules/monitoring-silencing-alerts.adoc:68]:
```suggestion
. On the *Create silence* page, set the schedule, duration, and label details for an alert.
```

File: modules/monitoring-silencing-alerts.adoc [hunk 15]
```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
...
++
+[NOTE]
+====
+You must add a comment before saving a silence.
+====
+
+. To save the silence, click *Silence*.
 
-. Expand the details for an alert by selecting *>* to the left of the alert name. Select the name of the alert in the expanded view to open the *Alert Details* page for the alert.
+To silence a set of alerts by creating a silence configuration in the *Administrator* perspective:
```
Comment (juzhao) [inline modules/monitoring-silencing-alerts.adoc:62]: the format should be like
```
To silence a et of alerts by creating a silence configuration :

-     In the Administrator perspective:

```

File: modules/monitoring-silencing-alerts.adoc [hunk 16]
```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
...
+[NOTE]
+====
+You must add a comment before saving a silence.
+====
 
-. To create the silence, select *Silence*.
+. To create silences for alerts that match the labels that you entered, click *Silence*.
 
-To silence a set of alerts by creating an alert specification in the *Administrator* perspective:
+To silence a set of alerts by creating a silence configuration in the *Developer* perspective:
```
Comment (juzhao) [inline modules/monitoring-silencing-alerts.adoc:77]: see from https://github.com/openshift/openshift-docs/pull/63944/files#r1312440037, change to below format is fine
```
   In the Developer perspective:

```

File: modules/monitoring-silencing-alerts.adoc [hunk 17]
```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
...
 
-. Navigate to the *Observe* -> *Alerting* -> *Silences* page in the {product-title} web console.
+. Go to *Observe* -> *<project_name>* -> *Silences* in the {product-title} web console.
 
-. Select *Create Silence*.
+. Click *Create Silence*.
```
Comment (juzhao) [inline modules/monitoring-silencing-alerts.adoc:81]:
```suggestion
. Click *Create silence*.
```

File: modules/monitoring-silencing-alerts.adoc [hunk 18]
```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
...
 
-. Set the schedule, duration, and label details for an alert in the *Create Silence* form. You must also add a comment for the silence.
+. On the *Create Silence* page, set the schedule, duration, and label details for an alert.
```
Comment (juzhao) [inline modules/monitoring-silencing-alerts.adoc:83]:
```suggestion
. On the *Create silence* page, set the duration, and label details for an alert.
```

File: modules/monitoring-silencing-alerts.adoc (hunk 16 above)
Comment (bburt-rh) [reply modules/monitoring-silencing-alerts.adoc:77]: Fixed

**Comment by rh-tokeefe** [comment]
LGTM

**Pull request merged**